### DISCUSSION TITLE
How to paste into zellij?

### DISCUSSION THREAD

**Discussion by vekexasia** [wezterm/wezterm#5461] [Q&A] [answered]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

It seems i am not able to paste into zellij.

What i tried: 

- use windows terminal + ctrl+v -> ok
- paste into wezterm via middle mouse -> ok
- paste into wezterm via ctrl +v -> no
- paste into zellij session from wezterm (both mouse and ctrlv) -> no

**Comment by nekowinston** [comment, marked as answer]
The [default key assignment](https://wezfurlong.org/wezterm/config/default-keys.html?h=pastefrom+clipboard) to paste from clipboard in WezTerm is Ctrl+Shift+V on Windows & Linux.
I can't replicate your exact setup, since I'm on macOS, but using WezTerm's paste function works inside zellij for me.
